packages/firestore/src/local/memory_remote_document_cache.ts

```diff
@@ -22,10 +22,9 @@ import {
 } from '../model/collections';
 import { Document, MaybeDocument } from '../model/document';
 import { DocumentKey } from '../model/document_key';
-
-import { PersistenceTransaction } from './persistence';
 import { PersistencePromise } from './persistence_promise';
 import { RemoteDocumentCache } from './remote_document_cache';
+import { PersistenceTransaction } from './persistence';
```

Comment:
Hrm. Does prettier not order these? (I'm not sure why this moved here)

Reply:
I think I asked vscode to organize imports. And I guess prettier didn't object either way? I'll put it back and see what happens.